modules/idaithalam/src/test/java/io/virtualan/idaithalam/core/generator/PostmanTests.java

```diff
@@ -19,25 +19,26 @@
 
     @Test
     public void addApikey() throws Exception {
+        final String yamlFile = "work-flow-apikey.yaml";
         Yaml yaml = new Yaml(new Constructor(ExecutionPlanner.class));
         InputStream inputStream = VirtualanTestPlanExecutor.class.getClassLoader()
-                .getResourceAsStream("work-flow-apikey.yaml");
+                .getResourceAsStream(yamlFile);
         //Check reading apiHeader
         ExecutionPlanner executionPlanner = yaml.load(inputStream);
         List<ApiExecutorParam> apiExecutor = executionPlanner.getApiExecutor();
-        Assert.assertTrue(apiExecutor.size() > 0);
+        Assertions.assertTrue(apiExecutor.size() > 0);
 
-        ApiExecutorParam apiExecutorParam1 = apiExecutor.get(0);
-        List<Map<String, Object>> apiHeaderList = apiExecutorParam1.getApiHeader().getHeaderList();
-        Assert.assertNotNull(apiHeaderList);
-        boolean checkApikey = false;
-        for (Map<String, Object> map : apiHeaderList) {
-            checkApikey = checkApikey || (map.get("X-API-KEY") != null && map.get("X-API-KEY").toString().equals("abc123"));
-        }
-        Assert.assertTrue(checkApikey);
+//        ApiExecutorParam apiExecutorParam1 = apiExecutor.get(0);
```

Comment:
remove the unused code

Reply:
It is only temporary unused.
I need it later.
Will correct that then.